libzim/libzim.pyx

```diff
@@ -190,7 +190,6 @@ class Compression(enum.Enum):
     # We don't care of the exact value. The function comp_from_int will do the right
     # conversion to zim::Compression
     none = 0
-    lzma = 1
     zstd = 2
```

Comment:
Should we change zstd value to 1 (and adapt `comp_from_int`) ?
If we keep the value 2, I'm afraid that we will think in the future that the value is meaningful and it is not.